getting-started.ipynb

```diff
@@ -0,0 +1,528 @@
+   "cell_type": "markdown",
+   "source": [
+    "Welcome to **PyTorch-Ignite**’s quick start guide that covers the\n",
+    "essentials of getting a project up and running while walking through\n",
+    "basic concepts of Ignite. In just a few lines of code, you can get your\n",
+    "model trained and validated. The complete code can be found at the end\n",
+    "of this guide."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    "This tutorial assumes you are familiar with the:\n",
+    "\n",
+    "1.  Basics of Python and deep learning\n",
+    "2.  Structure of PyTorch code"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Installation\n",
+    "\n",
+    "1.  From `pip`\n",
+    "\n",
+    "``` shell\n",
+    "pip install pytorch-ignite\n",
+    "```\n",
+    "\n",
+    "1.  From `conda`\n",
+    "\n",
+    "``` shell\n",
+    "conda install ignite -c pytorch\n",
+    "```\n",
+    "\n",
+    "See [here](/docs/how-to-guides/installation) for other installation\n",
+    "options."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Code\n",
+    "\n",
+    "Import the following:"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "source": [
+    "import torch\n",
+    "import torch.nn.functional as F\n",
+    "from torch import nn\n",
+    "from torch.optim import SGD\n",
+    "from torch.utils.data import DataLoader\n",
+    "from torchvision.datasets import MNIST\n",
+    "from torchvision.transforms import Compose, Normalize, ToTensor\n",
+    "from tqdm import tqdm\n",
+    "\n",
+    "from ignite.engine import Engine, Events, create_supervised_trainer, create_supervised_evaluator\n",
+    "from ignite.metrics import Accuracy, Loss"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Define a class of your model (or use the CNN model below) and then\n",
+    "instantiate it:"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "source": [
+    "class Net(nn.Module):\n",
+    "\n",
+    "    def __init__(self):\n",
+    "        super(Net, self).__init__()\n",
+    "\n",
+    "        self.conv1 = nn.Conv2d(1, 10, kernel_size=5)\n",
+    "        self.conv2 = nn.Conv2d(10, 20, kernel_size=5)\n",
+    "        self.conv2_drop = nn.Dropout2d()\n",
+    "        self.fc1 = nn.Linear(320, 50)\n",
+    "        self.fc2 = nn.Linear(50, 10)\n",
+    "\n",
+    "    def forward(self, x):\n",
+    "        x = F.relu(F.max_pool2d(self.conv1(x), 2))\n",
+    "        x = F.relu(F.max_pool2d(self.conv2_drop(self.conv2(x)), 2))\n",
+    "        x = x.view(-1, 320)\n",
+    "        x = F.relu(self.fc1(x))\n",
+    "        x = F.dropout(x, training=self.training)\n",
+    "        x = self.fc2(x)\n",
+    "\n",
+    "        return F.log_softmax(x, dim=-1)\n",
```

Comment:
Maybe, we can take a model from torchvision, slightly adapt few layers and do not bother with the complete definition etc

Reply:
Used a modified ResNet 18 along with changes in optimizer and loss function.